mlflow/pyspark/ml/log_model_allowlist.txt

```diff
@@ -0,0 +1,23 @@
+# classification
+pyspark.ml.classification.LinearSVC
+pyspark.ml.classification.DecisionTreeClassifier
+pyspark.ml.classification.GBTClassifier
+pyspark.ml.classification.LogisticRegression
+pyspark.ml.classification.RandomForestClassifier
+pyspark.ml.classification.NaiveBayes
```

Comment:
We do not auto log model for OneVsRest, because it may save multiple sub-models. and bring overhead.